pkg/statedb/db_test.go

```diff
@@ -91,517 +89,504 @@ func testWithDB(t testing.TB, withTags bool, test func(db *DB, table Table[testO
 	require.NoError(t, h.Start(context.TODO()))
 	t.Cleanup(func() {
 		assert.NoError(t, h.Stop(context.TODO()))
+		logging.SetLogLevel(logrus.InfoLevel)
 	})
-	test(db, table)
+	return db, table
 }
 
 func TestDB_LowerBound_ByRevision(t *testing.T) {
```

Comment:
Could some of these be https://pkg.go.dev/testing#T.Parallel? 
(Was reminded about this due to the Cleanup thing, https://brandur.org/fragments/go-prefer-t-cleanup-with-parallel-subtests)

Reply:
Yes they can. I'll play with it.

Reply:
Done. `0.346s` to `0.210s`.